wodles/aws/aws_s3.py

```diff
@@ -3113,6 +3149,7 @@ def main(argv):
     # Parse arguments
     options = get_script_arguments()
 
```

Comment:
Add the `iam_role_arn` validation from the `arg_valid_iam_role_duration` here:
```suggestion
if duration is not None and iam_role_arn is None:
    raise argparse.ArgumentTypeError('Used --iam_role_duration argument but no --iam_role_arn provided.')
```